cmd/mirroring/main.go

```diff
@@ -189,7 +195,7 @@ func main() {
 
 	for {
 		// Check for root hash consistency
-		newSTH, err := mirroring.VerifyLogConsistency(rekorClient, int64(sth.Size), sth.Hash)
+		newSTH, err := verify.VerifyCurrentCheckpoint(context.Background(), rekorClient, verifier, sth)
```

Comment:
This does the `sth.Verify`, so not sure if you need this above: if you remove it above, add a comment here saying it verifies the current checkpoint and verifies consistency against a newly fetched Checkpoint

Reply:
Whoops, I merged in the other PR which merged this commit. I'll update this!